src/main/java/org/reactome/addlinks/referencecreators/NCBIGeneBasedReferenceCreator.java

```diff
@@ -9,7 +11,14 @@
 public abstract class NCBIGeneBasedReferenceCreator extends SimpleReferenceCreator< Map<String,List<String>> >
 {
 	protected List<EntrezGeneBasedReferenceCreator> entrezGeneReferenceCreators;
-	
+
+	public List<EntrezGeneBasedReferenceCreator> getSubCreators()
+	{
+		return this.entrezGeneReferenceCreators != null
+				? Collections.unmodifiableList(this.entrezGeneReferenceCreators)
+				: Collections.unmodifiableList(new ArrayList<EntrezGeneBasedReferenceCreator>());
```

Comment:
For the last part of the ternary, you could say `Collections.emptyList()`.  I think the compiler should be able to infer the type and `emptyList()` will return an immutable list: https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#emptyList--